lightning/src/ln/channel.rs

```diff
@@ -2761,6 +2771,11 @@ impl<Signer: Sign> Channel<Signer> {
 			*self.next_remote_commitment_tx_fee_info_cached.lock().unwrap() = None;
 		}
 
+		self.holder_signer.validate_counterparty_revocation(
+			self.cur_counterparty_commitment_transaction_number + 1,
+			&secret
+		).map_err(|_| ChannelError::Close("Failed to validate your revocation".to_owned()))?;
```

Comment:
s/your/counterparty

Reply:
I'm not sure about this one.  This is a message we are sending to the counterparty, so wouldn't it be confusing to them that we are referring to them as "counterparty" instead of "you"?

Reply:
Actually, elsewhere the term "Peer" is used, so I'll use that.